WalletWasabi/WabiSabi/Backend/Rounds/RoundParameters.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace WalletWasabi.WabiSabi.Backend.Rounds;
 
-public record RoundParameters
+public record RoundParameters : IUtxoSelectionParameters
```

Comment:
No. RoundParameters has nothing to do with IUtxoSelectionParameters, those are two completely different concepts that only share some fields.